README.md

```diff
@@ -276,6 +276,7 @@ Values are optional and default to the values provided on the command-line or de
 A scenario can define changes that should be applied to the source before each build. You can use this to benchmark or profile an incremental build. The following mutations are available:
 
 - `apply-build-script-change-to`: Add a statement to a Groovy or Kotlin DSL build script, init script or settings script. Each iteration adds a new statement and removes the statement added by the previous iteration.
+- `apply-dependency-change-to`: Add dependencies as projects to a Groovy or a Kotlin DSL build script. Each iteration adds a new combinations of projects as dependencies and removes the projects added by the previous iteration.
```

Comment:
I think the name should be reflective to the fact that we are modifying _project_ dependencies. Something like `--apply-project-dependency-change-to`?

Reply:
Agreed. I would be happy to get a suggestions also for:
```
            projects-set-size = 10
            applied-projects-size = 5
```
since it feels like they don't tell exactly what they do (I already renamed that 2 times but I am still not happy)

Reply:
Renamed, also renamed `applied-projects-size` to `applied-projects-set-size`